src/crimson/net/Socket.h

```diff
@@ -22,6 +24,9 @@ class Socket
   seastar::input_stream<char> in;
   seastar::output_stream<char> out;
 
+  bool down = false;
```

Comment:
shall we remove these two member variables? i think they are only for debugging purpose. if yes, we need to guard them with `#ifdef NDEBUG`.

Reply:
agree, fixed.